adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1185,7 +1190,8 @@ private case class GenericGenomicRDD[T](
     IntervalArray(rdd)
   }
 
-  protected def getReferenceRegions(elem: T): Seq[ReferenceRegion] = {
+  protected def getReferenceRegions(elem: T,
+                                    stranded: Boolean = false): Seq[ReferenceRegion] = {
```

Comment:
In this function, `regionFn` should be updated to take `stranded` as a parameter, no?